sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1932,6 +1963,9 @@ export interface SearchIndexer {
     targetIndexName: string;
 }
 
+// @public
+export type SearchIndexerAlias = SearchAlias;
```

Comment:
why have both `SearchAlias` and `SearchIndexerAlias`? Seems like we only need one?

Reply:
Actually, it should be `SearchIndexAlias`. I will fix that. But, why do we have both? The reason is I have renamed the SearchAlias to SearchIndexAlias to be in sync with other naming conventions we followed in the SDK. 

And both has to be exposed in the index file. Else, API Extractor will complain about a missing export.